src/native-common/AccessibilityUtil.ts

```diff
@@ -9,8 +9,15 @@
 */
 
 import _ = require('./lodashMini');
+import assert = require('assert');
+import React = require('react');
+import RN = require('react-native');
 
 import { AccessibilityUtil as CommonAccessibilityUtil } from '../common/AccessibilityUtil';
+import AndroidAccessibilityUtil from '../android/AccessibilityUtil';
```

Comment:
While this is an anti-pattern, it's beneficial to us.  The idea behind this is that pretty much every UI component needs to talk to AccessibilityUtils in some way, shape or form.  That leaves a few options:

1) Extend every UI component (that needs accessibility) for every platform so that it imports the platform specific AccessibiilityUtils and calls the function
2) Provide some sort of override/shim at the RX platform creation (something AccessibilityUtils.setPlatformHelperFunctions that pipes to this._impl that's in this PR)
3) Have native-common/AccessibilityUtils decide which impl to use on init and import platform specific implementations

#1 adds a bunch of extra code and another layer of abstract classes
#2 is somewhat obscure and non-obvious to someone working in the codebase
#3 isn't ideal but seems somewhat better than the other options.

Do you have any alternative solutions?